### PR TITLE
Clarify current behavior of documents

### PR DESCRIPTION
**Pull request by XanderWA**

### CODE REVIEW

File: user-guide/Basic_Functionality/Documents/Managing_the_document_folders.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Within each folder, you can **add, rename, or delete custom folders** through th
 
 On the *Documents* page of an element, there is also a default *Elements* folder containing a subfolder for that specific element. This subfolder contains documents that are associated with that particular element only. It is not possible to add or rename such element folders.
 
-The documents in such an element folder are stored on the DataMiner Agent hosting the element, and they are not synced with other Agents in the cluster. This makes this folder especially useful for storing larger files.
+The documents in such an element folder are stored on the DataMiner Agent hosting the element, and they are not synced with other Agents in the cluster. This makes this folder especially useful for storing larger files. These documents can't be opened through Cube.
+
+Note that these files are synced to a failover buddy *if* the file size doesn't exceed the 20MB syncing limit.
```
Comment (bertbsky) [inline user-guide/Basic_Functionality/Documents/Managing_the_document_folders.md:27]: Maybe reword as a warning:
> [!WARNING]
> On a failover pair, element documents are only synced between the agents if they don't exceed the filesize limit  (default: 20MB). This means that after a failover switch, the element may no longer have access to its own large documents.

**Comment by MariekeGO** [comment]
@XanderWA, I've reviewed your PR and made some changes. Is everything still OK for you?

**Comment by XanderWA** [comment]
Looks great! Thanks!

**Review by MariekeGO — APPROVED** [review]
